src/gameStateMachine.ts

```diff
@@ -0,0 +1,286 @@
+import { createMachine, assign } from "xstate";
+
+interface Artist {
+    id: number;
+    name: string;
+}
+
+interface Lyric {
+    text: string;
+    referentId: number;
+}
+
+interface Lyrics {
+    warped: Lyric[];
+}
+
+interface Song {
+    id: number;
+    title: string;
+    artistName: string;
+    lyrics: Lyrics;
+}
+
+export interface GameContext {
+    totalGuesses: number;
+    correctGuesses: number;
+    artistList: Array<Artist>;
+    selectedArtist?: Artist;
+    selectedSong?: Song;
+}
+
+export type GameEvent =
+    | {
+          type: "START";
+      }
+    | {
+          type: "SELECT_ARTIST";
+          artist: Artist;
+      }
+    | {
+          type: "CLEAR_SELECTED_ARTIST";
+      }
+    | {
+          type: "RESOLVE";
+      }
```

Comment:
You mentioned removing this one, have you just not pushed your latest?

Reply:
ah, I removed it from the machine, but didn't remove the typing for the event.